src/main/java/com/github/s4u/plugins/PGPVerifyMojo.java

```diff
@@ -111,11 +111,11 @@
      *
      * @since 1.0.0
      */
-    @Parameter(property = "pgpverify.keyserver", defaultValue = "hkp://pool.sks-keyservers.net", required = true)
+    @Parameter(property = "pgpverify.keyserver", defaultValue = "hkps://pgp.mit.edu/", required = true)
```

Comment:
I would like to stay with pool.sks-keyservers.net when we will have implementation for support their CA we can change defaults to hkps.pool.sks-keyservers.net. 
You can override this param in your configuration.